frontend/src/libs/components/stepper/stepper.tsx

```diff
@@ -0,0 +1,90 @@
+import { Button, Icon } from '~/libs/components/components.js';
+import { IconName } from '~/libs/enums/enums.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useMemo, useStepper } from '~/libs/hooks/hooks.js';
+
+import styles from './styles.module.scss';
+
+const ONE_STEP_LENGTH = 1;
+
+type Properties = {
+  children: React.ReactNode[];
+  className?: string;
+};
+
+const Stepper: React.FC<Properties> = ({ children, className }: Properties) => {
+  const {
+    currentStep,
+    isFirstStep,
+    isLastStep,
+    buttonLabel,
+    barYellowWidth,
+    toNextStep,
+    toPreviousStep,
+  } = useStepper({ length: children.length });
+
+  const stepBlock = useMemo(() => {
+    return children[currentStep - ONE_STEP_LENGTH];
+  }, [children, currentStep]);
+
+  return (
+    <div className={getValidClassNames(styles.wrapper, className)}>
+      <div className={styles.barWrapper}>
+        <div className={styles.bar}>
+          <div
+            className={getValidClassNames(styles.bar, styles.barYellow)}
+            style={{ width: `${barYellowWidth}%` }}
+          ></div>
+        </div>
+
+        <div className={styles.stepperWrapper}>
+          <div className={styles.imgWrapper}>
+            <Icon
+              iconName={IconName.ARROW}
+              className={getValidClassNames(
+                styles.img,
+                isFirstStep && styles.imgDisabled,
+              )}
+              onClick={toPreviousStep}
+            >
+              <span className="visually-hidden">Go to the previous step</span>
+            </Icon>
+            <Icon
+              iconName={IconName.ARROW}
+              className={getValidClassNames(
+                styles.img,
+                styles.imgRight,
```

Comment:
Let's better remove this style and add to separate icons for `arrow-left` and `arrow-right`, because now it's a little bit confusing to have `arrow-left` as `arrow` and then rotating it